include/c#/10/OfficialAPI/APICache.cs

```diff
@@ -66,9 +65,9 @@ public static async ValueTask<SkillId> ResolveWeaponSkill(BuildCode code, Weapon
 		}
 
 		var skill = (code.Profession) switch {
-			Profession.Thief        => weapon.Skills.FirstOrDefault(skill => skill.Slot == $"Weapon_{skillIndex + 1}" && (skillIndex != 2 || skill.Offhand == effectiveWeapons.OffHand)),
+			Profession.Thief        => weapon.Skills.Find(skill => skill.Slot == $"Weapon_{skillIndex + 1}" && (skillIndex != 2 || skill.Offhand == effectiveWeapons.OffHand)),
```

Comment:
There is no values in changing this. Since there are no downsides aswell ill just leave it as is, but please avoid changes like this in the future. I would argue that using `FirstOrDefault` here is the better choice because it clearly states what the function does. Even i had to make sure that `Find` actually handles the 'not found' case in the same way.